Evaluations.js

```diff
@@ -193,47 +193,60 @@ function generateReviewMatrix() {
       return;
     }
 
-    // Получаем email submitters
+    // Getting email of submitters
     const submittersEmails = validResponses.map((row) => row[1]); // Assuming Email is in column 2
     Logger.log(`Submitters Emails: ${JSON.stringify(submittersEmails)}`);
 
-    // Получаем все emails из Registry
+    // Get all ambassador emails from the registry
     const allAmbassadorsEmails = registrySheet
       .getRange(2, 1, registrySheet.getLastRow() - 1, 1)
       .getValues()
       .flat();
     Logger.log(`All Ambassadors Emails: ${JSON.stringify(allAmbassadorsEmails)}`);
 
-    // Назначаем evaluators submitters
-    const evaluatorQueue = [...allAmbassadorsEmails];
-    evaluatorQueue.sort(() => Math.random() - 0.5); // Shaffle evaluators pool
-    Logger.log('Shuffled evaluator pool.');
+    // Create a pool of potential evaluators, allowing each evaluator to be picked up to 3 times
+    const potentialEvaluators = [...allAmbassadorsEmails, ...allAmbassadorsEmails, ...allAmbassadorsEmails];
```

Comment:
These should be filtered for expelled. Only active ambassadors can review.

Reply:
> As for simply removing the expelled ambassador from all lists – that would be completely unaccountable.

I didn't communicate very clearly. When I said to filter them, I mean from the evaluators list on line 207. The expelled ambassadors must be removed from the potential evaluators.

Reply:
Of course! This part is not finished until we come to an agreement regarding everything about the expelled status! Essentially, it is not finished

Reply:
i have a question: why didn't you make eslint hidden (with dot) as .prettirrc? Though it anyway ignored because of it's type .mjs, just curious

> I didn't communicate very clearly. When I said to filter them, I mean from the evaluators list on line 207. The expelled ambassadors must be removed from the potential evaluators.

Ahh .. this is temporary, for testing purposes